ceph-salt-formula/salt/macros.yml

```diff
@@ -26,17 +26,14 @@
 {%- endmacro %}
 
 {% macro sudoers(name) -%}
-{%- set ssh_user = pillar['ceph-salt']['ssh']['user'] %}
-{% if ssh_user != 'root' %}
 {{name}}:
   file.append:
-    - name: /etc/sudoers.d/{{ssh_user}}
+    - name: /etc/sudoers.d/cephadm
```

Comment:
You know, we could avoid the collision with the `/etc/sudoers.d/cephadm` installed by Ceph by simply using a different filename - e.g. `/etc/sudoers.d/ceph-salt`, or `/etc/sudoers.d/cephadm-ceph-salt`

Reply:
Good catch! Thanks!